Makefile

```diff
@@ -0,0 +1,34 @@
+# Convenience Makefile for generating and releasing SBP client
+# libraries.
+
+SWIFTNAV_ROOT := $(shell pwd)
+MAKEFLAGS += SWIFTNAV_ROOT=$(SWIFTNAV_ROOT)
+SBP_SPEC_DIR := $(SWIFTNAV_ROOT)/spec/yaml/swiftnav/sbp/
+SBP_GEN_BIN := python $(SWIFTNAV_ROOT)/sbpg/generator.py
+
+help:
+	@echo "Please use \`make <target>' where <target> is one of"
+	@echo "  all       to make SBP clients across all languages"
+	@echo "  c         to make C headers"
+	@echo "  python    to make Python bindings"
+	@echo "  docs      to make LaTeX documentation"
+
+all: c python docs
+
+c:
+	@echo "Generating C headers..."
+	$(SBP_GEN_BIN) -i $(SBP_SPEC_DIR) \
+		       -o $(SWIFTNAV_ROOT)/c/include/libsbp \
+	               --c;
+
+python:
+	@echo "Generating Python bindings..."
```

Comment:
The first rule is the default target (if you just type make) - put the all rule first? The echo for all will be *after* the c and python and docs have been gen'd.

Add some [PHONY targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)?

```make
.PHONY: all c python docs
```